app/src/features/admin/users/ManageUsersPage.tsx

```diff
@@ -112,6 +113,30 @@ const ManageUsersPage: React.FC = () => {
     getCodes();
   }, [biohubApi.codes, isLoadingCodes, codes]);
 
+  const sendEmail = () => {
```

Comment:
If this is just for testing purposes, I would remove it from your PR.

Also, for future when we have real content for SIMS notifications, it would be good to move any pre-canned messages into a constants file for notifications.  So later on we can do something like `sendGCNotification(recipient, GC_NOTIFICATIONS.SIGNUP_SUCCESS_NOTIFICATION)`